### DISCUSSION TITLE
Script executor example

### DISCUSSION THREAD

**Discussion by mikch06** [zauberzeug/nicegui#974] [Q&A] [answered]
### Question

The one and only documentation which i am missing is the script_executor :D 
I like to run a python script or shell script by nicegui. But don't find how to. 

Thx in advance.

**Comment by mikch06** [comment, marked as answer]
think i got it, sorry.

**Reply by rodja** [reply]
Just for reference: https://github.com/zauberzeug/nicegui/blob/main/examples/script_executor/main.py

**Reply by mikch06** [reply]
Yes, Thanks! Didn't know first how to implement!
